install_config/aggregate_logging.adoc

```diff
@@ -877,11 +877,16 @@ interpret messages from
 link:https://www.elastic.co/guide/en/logstash/current/plugins-codecs-fluent.html[Fluentd]
 and can be configured to use
 link:https://www.elastic.co/guide/en/logstash/current/plugins-inputs-tcp.html#ssl_enable[SSL].
+Currently however, the `secure_forward` plugin is required to provide a `shared_key`
+which Logstash is unable to interpret and provide. In order to forward logs from
+Fluentd to Logstash the `forward` Fluentd plugin will need to be used instead.
 
 Further explanation of how to set up the `fluent-plugin-secure-forward` plug-in
 can be link:https://github.com/tagomoris/fluent-plugin-secure-forward[found
 here].
 
+Further explanation of how to set up the `forward` plug-in can be link:http://docs.fluentd.org/articles/out_forward[found here].
```

Comment:
Consider rewording to:
link:http://docs.fluentd.org/articles/out_forward[Learn more about the `forward` plug-in.]

or something like that :thinking: